Bresenham_Algo_Programs/Line drawing algo.java

```diff
@@ -0,0 +1,104 @@
+package applet;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseMotionListener;
+
+public class bresenhamLine extends java.applet.Applet implements MouseListener, MouseMotionListener {
+ 
+ private static final long serialVersionUID = 1L;
+ int width, height;
+ int xa = 0, ya = 0, xb = 0, yb = 0, pixelsize = 2;
+
+ public void init() {
+       width = getSize().width;
+       height = getSize().height;
+       this.addMouseListener(this);
+        this.addMouseMotionListener(this);
+    
+    }
+
+ // draw a line from point xa,ya into xb,yb
+ public void bresenham_line(int xa, int ya, int xb, int yb) {
+
+  // if point xa, ya is on the right side of point xb, yb, change them
+  if ((xa - xb) > 0) { bresenham_line(xb, yb, xa, ya); return; }
+
+  // test inclination of line
```

Comment:
If you are adding comments then please try following the standard commenting practice. 
For methods mention the parameter, return type in  a multiline comment